examples/experimental/dagster-airlift/setup.py

```diff
@@ -19,7 +19,7 @@ def get_version() -> str:
 
 AIRFLOW_REQUIREMENTS = [
     # Requirements for python versions under 3.12.
-    "apache-airflow>=2.0.0; python_version < '3.12'",
```

Comment:
?

Reply:
I'm running into non-determinism with different versions of airflow. To get a consistent test setup between local and bk, I'm pinning airflow in tests for now.